docs/tfengine/schemas/resources.md

```diff
@@ -186,6 +186,11 @@
 | healthcare_datasets.iam_members.role | IAM role to grant. | string | true | - | - |
 | healthcare_datasets.name | Name of healthcare dataset. | string | true | - | - |
 | iam_members | Map of IAM role to list of members to grant access to the role. | object | false | - | - |
+| kubernetes_service_accounts | Kubernetes service accounts. See <https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/service_account>. | array() | false | - | - |
+| kubernetes_service_accounts.google_service_account_email | Email of the google service account the kubernetes service account should use to authenticate with other resources. | - | true | - | - |
+| kubernetes_service_accounts.name | Name of the KSA. | string | true | - | - |
```

Comment:
Nit: You use "KSA" without defining it. Maybe above on line 189 have "Kubernetes Service Accounts (KSAs)".

Reply:
Done.